vllm/v1/engine/output_processor.py

```diff
@@ -106,6 +112,7 @@ def abort_requests(
     def add_request(
         self,
         request: EngineCoreRequest,
+        lora_request: Optional[LoRARequest] = None,
```

Comment:
same comment as above `EngineCoreRequest.lora_request` has this value already

Reply:
Fixed